hackathon/views.py

```diff
@@ -177,6 +177,7 @@ def create_hackathon(request):
         # Submit form and save record
         if form.is_valid():
             form.instance.created_by = request.user
+            form.instance.organiser = request.user
```

Comment:
Is the `organiser` meant to be the person creating the Hackathon, or isn't that meant to be `CI` or one of the `ETBs` for example? I thought organizer was meant to be the organization the Hackathon is being held for?
If so, should this actually be `request.user` or the ORG name?

Reply:
`organiser` is kind of a duplicate of `created_by`, the field you are talking about is `organisation`.